lib/Conversion/ImportVerilog/Types.cpp

```diff
@@ -192,6 +192,11 @@ struct TypeVisitor {
                                           StringAttr{}, loc);
   }
 
+  // Handle void types.
```

Comment:
Nice! Feel free to directly PR this upstream together with a parsing test that exercises this code path. Maybe Slang will let you do something weird like a void variable `void x;`. Or otherwise you could try `{0{y}}` as you mentioned. It's good to have these small PRs that add independent features, together with tests that ensure they work as intend (and don't accidentally break down the road when others make changes to the code).

Reply:
Thanks for your reminder!  I'll be careful to push my code. I have to learn more about `void`, then land it. 
`{0{y}}` is also a `void` type.
# .sv
```
module Foo;
  parameter p = 32;
  bit a = {{0{1'b1}},{p-32{1'b0}},1'b1};
endmodule
```
# Json
``` json
{
                  "kind": "Replication",
                  "type": "void",
                  "count": {
                    "kind": "IntegerLiteral",
                    "type": "int",
                    "value": "0",
                    "constant": "0"
                  },
                  "concat": {
                    "kind": "Concatenation",
                    "type": "bit[0:0]",
                    "operands": [
                      {
                        "kind": "IntegerLiteral",
                        "type": "bit[0:0]",
                        "value": "1'b1",
                        "constant": "1'b1"
                      }
                    ],
                    "constant": "1'b1"
                  },
                  "constant": "null"
                },
```